script/core.hbs

```diff
@@ -38,12 +38,12 @@
       ...token && { apiKey: token },
       querySource: window.isOverlay ? 'OVERLAY' : 'STANDARD',
       onStateChange: (objParams, stringParams, replaceHistory) => {
-        if ('parentIFrame' in window) {
+        window.iframeLoaded.then(() => {
```

Comment:
is there some chance of this being a memory leak, if window.iframeLoaded is never fulfilled? (i.e. we're on a non-iframe integration)

Reply:
despite onStateChange running for literally every SDK storage update I would think the memory still wouldn't matter, but hopefully there would be a relatively easy fix?

Reply:
note: Connor thought of this first I just parroted it